packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -571,7 +571,8 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 				statusbar: false,
 				target_list: false,
 				table_resize_bars: false,
-				language: ['en_US', 'en_GB'].includes(language) ? undefined : language,
+				language_url: `build/lib/tinymce/langs/${['en_US', 'en_GB'].includes(language)}` ? undefined : language,
```

Comment:
Need to fix this - the whole url should be undefined if the language is not supported.